config/babel.dev.js

```diff
@@ -10,13 +10,15 @@
 // @remove-on-eject-end
 
 var path = require('path');
+var findCacheDir = require('find-cache-dir');
 
 module.exports = {
   // Don't try to find .babelrc because we want to force this configuration.
   babelrc: false,
   // This is a feature of `babel-loader` for webpack (not Babel itself).
-  // It enables caching results in OS temporary directory for faster rebuilds.
-  cacheDirectory: true,
+  // It enables caching results in ./node_modules/.cache/create-react-app/
+  // directory for faster rebuilds.
+  cacheDirectory: findCacheDir({ name: 'create-react-app' }),
```

Comment:
The convention seems to be that this should be the project name from package.json. Since this file is used by `react-scripts`, should we use `react-scripts` as the name here?